src/routes/submit/submit.js

```diff
@@ -0,0 +1,47 @@
+import {
+  sendEmailToApplicant,
+  sendEmailToCaseWorker
+} from '../../common/connectors/notify/notify.js'
+import { getApplicationReference } from '../../common/helpers/application-reference/index.js'
+import { getQuestionFromSections } from '../../common/helpers/data-extract/data-extract.js'
+import { generateEmailContent } from '../../common/helpers/email-content/email-content.js'
+import { isValidPayload, isValidRequest } from './submit-validation.js'
+
+export const submit = [
+  {
+    method: 'POST',
+    path: '/submit',
+    handler: async (request, h) => {
+      if (!isValidRequest(request)) {
+        return h.response({ error: 'Invalid request' }).code(400)
```

Comment:
The other tickets have the error code in constant case - e.g. "INVALID_REQUEST" or "INVALID_PAYLOAD".  I think it's useful to stick to this convention, because it tells the recipient that that's a part of the API they can depend on / dispatch on.

Reply:
done